### DISCUSSION TITLE
Live grep at nvim-tree (or persistent window with search)

### DISCUSSION THREAD

**Discussion by dro-sh** [ibhagwan/fzf-lua#1562] [Q&A] [answered]
I want to have persistent window for live grep. I find response at FAQ about [split layout](https://github.com/ibhagwan/fzf-lua/wiki#split-layout), but then I open it, write pattern and select file, window disappears and I should execute command again to open fzf search window, but cursor begins at first item of list.
The best feature will be live grep at nvim-tree and write there patterns to find needed files.

**Comment by ibhagwan** [comment, marked as answer]
No such option, unfortunately that would require design changes.
